### PR TITLE
docs: fix various typos & grammar mistakes

### PR DESCRIPTION
**Pull request by DomAlexRod**
Rectifying a small number of spelling and grammatical issues across the documentation.
Also added consistent comment formatting to [examples/pytorch-multigpu-example.py](https://github.com/mlco2/codecarbon/compare/master...DomAlexRod:codecarbon:master#diff-f7448b41f72dedfe1bb6725327338be0dc3b38261837e09ff6bd79ce44fee7fe). Comments now start with a captial letter as per PEP 8 standard.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ Then, click on the url displayed in the terminal.
...
 
-The coding style and linting rules are automatically applied and enforce by [pre-commit](https://pre-commit.com/). This tool helps to maintain the same code style across the code-base to ease the review and collaboration process. Once installed ([https://pre-commit.com/#installation](https://pre-commit.com/#installation)), you can install a Git hook to automatically run pre-commit (and all configured linters/auto-formatters) before doing a commit with `pre-commit install`. Then once you tried to commit, the linters/formatters will run automatically. It should display something similar to:
+The coding style and linting rules are automatically applied and enforce by [pre-commit](https://pre-commit.com/). This tool helps to maintain the same code style across the code-base and to ease the review and collaboration process. Once installed ([https://pre-commit.com/#installation](https://pre-commit.com/#installation)), you can install a Git hook to automatically run pre-commit (and all configured linters/auto-formatters) before doing a commit with `pre-commit install`. Then once you tried to commit, the linters/formatters will run automatically. It should display something similar to:
```
Comment (benoit-cty) [inline CONTRIBUTING.md:185]: Are you sure ? I think it's better without 'and', or add a comma ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -89,20 +89,20 @@ $ python -m unittest tests.test_your_feature.YourTestCase.test_function
...
 
-`nvidia-smi` is a usefull tool to see the metrics of the GPU and compare it with CodeCarbon.
+`nvidia-smi` is a useful tool to see the metrics of the GPU and compare it with CodeCarbon.
 
-### Versionning
+### Versioning
 
 
-To add a new feature to codecarbon, the following workflow is applied :
+To add a new feature to codecarbon, the apply the following workflow:
```
Comment (benoit-cty) [inline CONTRIBUTING.md:105]:
```suggestion
To add a new feature to codecarbon, apply the following workflow:
```

**Review by benoit-cty — COMMENTED** [review]
Thank you for this PR. Most contributors of CodeCarbon are French like me, so we make many mistakes.

**Review by benoit-cty — APPROVED** [review]
Thank you !

**Comment by DomAlexRod** [comment]
Apologies! I managed to close the PR and looks like I need a maintainer to run the Actions to be able to merge.

**Pull request merged**